lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -46,9 +65,19 @@ all_afs:
   config_get_token_append: '/^address-family (\S+) (\S+)$/'
 
 client_to_client:
-  config_get_token_append: '/^client-to-client reflection$/'
-  config_set_append: '<state> client-to-client reflection'
-  default_value: true
+  cli_nexus:
+    config_get_token_append: '/^client-to-client reflection$/'
+    config_set_append: '<state> client-to-client reflection'
+    default_value: true
+  cli_ios_xr:
+    config_set: 'router bgp <asnum>'
+    config_set_append:
+      - ' address-family <afi> <safi>'
+      - ' <state> bgp client-to-client reflection disable'
```

Comment:
As we discussed this morning - if this command is applicable under the default vrf only, you should check in bgp_af.rb when this property is being set and handle it correctly, something like:

```diff
    def client_to_client
+      return nil if platform == :ios_xr && @vrf != 'default'
      state = config_get('bgp_af', 'client_to_client', @get_args)
      state ? true : false
    end

    def client_to_client=(state)
+      fail Cisco::UnsupportedError, 'client-to-client is not configurable on a per-VRF basis on IOS XR' \
+        if platform == :ios_xr && @vrf != 'default'
      state = (state ? '' : 'no')
      set_args_keys(state: state)
      config_set('bgp_af', 'client_to_client', @set_args)
    end

    def default_client_to_client
+      return nil if platform == :ios_xr && @vrf != 'default'
      config_get_default('bgp_af', 'client_to_client')
    end

```

and should explicitly test for this behavior in the minitest.